request.go

```diff
@@ -981,3 +1022,80 @@ func (req *ExecuteRequest) Args(args interface{}) *ExecuteRequest {
 func (req *ExecuteRequest) Body(res SchemaResolver, enc *msgpack.Encoder) error {
 	return fillExecute(enc, req.expr, req.args)
 }
+
+// StickyRequest is an interface that provides the info about a Connection
+// the request belongs to.
+type StickyRequest interface {
+	Conn() *Connection
+}
```

Comment:
A good idea, but the description does not match the implementation. This is unexpected and confusing. `Sticky`confuses too.

```suggestion
// ConnectionRequest is an interface that provides the info about a Connection
// the request belongs to.
type ConnectionRequest interface {
        Request
	Conn() *Connection
}
```
Maybe other naming: `ConnectedRequest` ? We can think about it.

Alternatively, you can add an interface:
```Go
type XXX interface {
    Conn() *Connection
}
```
I'm not sure about the name, but definitely not `*Request`.

Reply:
Fixed, check it.